stable/drone/templates/ingress.yaml

```
@@ -1,7 +1,7 @@
 {{- if .Values.ingress.enabled -}}
 {{- $fullName := include "drone.fullname" . }}
 {{- $httpPort := .Values.service.httpPort }}
-apiVersion: networking.k8s.io/v1beta1
+apiVersion: {{ default "networking.k8s.io/v1beta1" .Values.ingress.apiVersion }}
```

How about this to make it compatible with versions <= k8s 1.13

```suggestion
{{ if .Capabilities.APIVersions.Has "networking.k8s.io/v1beta1" }}
apiVersion: networking.k8s.io/v1beta1
{{ else }}
apiVersion: extensions/v1beta1
{{ end -}}
```